src/overlays/actors/ovl_En_Clear_Tag/z_en_clear_tag.c

```diff
@@ -61,25 +76,1001 @@ static ColliderCylinderInit D_809D5C40 = {
     },
     { 15, 30, 10, { 0, 0, 0 } },
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Clear_Tag/func_809D35B0.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Clear_Tag/func_809D36A8.s")
+static u32 D_809D5C98 = 0; // unused
+static u32 D_809D5C9C = 0; // unused
+
+static EnClearTagEffect sClearTagEffects[CLEAR_TAG_EFFECT_MAX_COUNT];
+
+#include "../../../../assets/overlays/ovl_En_Clear_Tag/ovl_En_Clear_Tag.c"
```

Comment:
we dont use relative paths anymore in the project

Reply:
We can't remove the relative path here because include starts in the current directory. We would have to update extraction script to put the C file for overlays in the overlay's folder, rather than assets.

Reply:
```suggestion

#include "overlays/ovl_En_Clear_Tag/ovl_En_Clear_Tag.c"
```
Does this not work?

Reply:
Ah I had tried it from the project root, not the source root. This worked.